library/globals.inc.php

```diff
@@ -915,6 +915,12 @@ function gblTimeZones()
             'bool',                           // data type
             '1',                              // default
             xl('Observation Results in Immunization')
+        ),
+        'search_method_in_patient_list' => array(
+            xl('Search method in patient list'),
+            'bool',                           // data type
+            '1',                              // default
+            xl('Uncheck to enable exact search')
```

Comment:
Regarding the search method, what specifically are you trying to accomplish? This is an area where OpenEMR really needs some improvement and would be good to have a wish list on this to figure out the best route to go.

Reply:
Hi @bradymiller  , some of the users of OpenEMR asked that we will enable them to do an exact search and not search in the form of like. This request was particularity important in the patient finder . Important to mentioned that we use only the global search in that table 
and not the search per column which we hided for simpler use.   
the issue here is that if a person search for patient id "11"  and he gets all the ids (and there is a lot of them ) as following :  11 , 111, 1134535  and etc in addition to phone numbers that contain 11 in the same manner. to reduce that I added exact search .

Reply:
Rather than a global I think a criteria panel for per use would be a better way to go. Even if it is as simple as a checkbox in patient finder UI. To take to the next level it would be worth the effort to setup a per user persistence table to set defaults and context bindings for various actions. Much like we do for user settings edit.

Reply:
Would it make sense to also support the wildcard character * in the searches?

Reply:
Hi @bradymiller  do you approve @sjpadgett idea of adding checkbox instead of Global ?
I do not think wildcard is needed here because there is the other type of search (that can be easily switch if we implement the checkbox ) but if you think it would be nice I will add it.

Reply:
hi @eyalvo-matrix ,
Checkbox seems fine (note you can actually save the checkbox entry as a user setting (check out examples of the Open New Window toggle box on the Flow Board to see what I mean).
Agree wildcard is not needed for what you are doing